src/cdk/bootstrap.ts

```diff
@@ -17,7 +17,9 @@ import ChangeDetector from '../change-detector'
 import ErrorLogger from '../error-logger'
 import { quickDeploy, removeAssetDirectories } from './deploy-buckets'
 
-type ValidAwsContruct = CloudFormation
+type ValidAwsContruct = CloudFormation;
+
+const supportedCdkArgs = ['outputs-file', 'require-approval']
```

Comment:
Suggesting we could put this into a separate file that is solely responsible for parsing CLI args.